### DISCUSSION TITLE
브랜치 생성 관련

### DISCUSSION THREAD

**Discussion by kyu-baek** [3DPong/transcendence#17] [Q&A] [answered]
backend task 를 위해 브랜치를 만들려고 합니다. 
브랜치를 생성하고 싶은데 개발용 backend/develop 부터 우선 올려야 할 것 같습니다.
요청드립니다~

**Comment by JuneParkCode** [comment, marked as answer]
생성 완료했습니다. 
**commit : 1930adc**
